bin/node-template/runtime/Cargo.toml

```diff
@@ -13,6 +13,8 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 codec = { package = "parity-scale-codec", version = "1.3.4", default-features = false, features = ["derive"] }
 
+system-runtime-rpi = { default-features = false, package = "frame-system-rpc-runtime-api" , version = "2.0.0-rc5", path = "../../../frame/system/rpc/runtime-api/" }
```

Comment:
Is this a typo , or did you intentionly used rpi instead of api?

Reply:
Intentional, my way of condensing the name- I'll probably change it back to be more consistent with the rest of the project, my apologies, thanks!

Reply:
yeah, lets avoid renaming things :)